docs/local-content-items.md

```diff
@@ -0,0 +1,11 @@
+# Local Content Items
+
+For local development and preview apps, it is sometimes desirable to have the ability to load content items that are specified locally rather than pointing to production/integration or having the content store running locally.
+
+To support this, set:
+
+`ALLOW_LOCAL_CONTENT_ITEM_OVERRIDE=true`
```

Comment:
It'd be good if there was an accompanying change in govuk-docker that sets this.

Reply:
https://github.com/alphagov/govuk-docker/pull/793

Reply:
Done (and added it to app.json)